Logger.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+
+public protocol Logger {
+    
+    func logNetworkRequest(request: NSURLRequest) -> Void
```

Comment:
These probably don't need the explicit `-> Void`, do they? They could use some descriptive comments though.